js/plugins.js

```diff
@@ -539,8 +539,29 @@ plugins.factory('userPlugins', function() {
         }
     });
 
+    /*
+     * Streamable Embedded Player
+     */
+    var streamablePlugin = new Plugin('Streamable video', function(message) {
+        var rPath = /streamable\.com\/s?\/?(.+)/;
```

Comment:
this might match unrelated stuff with `streamable.com/blah` anywhere in a URL, it would probably be best to include the protocol explicitly.